e/elasticsearch/Dockerfiles/template/Dockerfile

```diff
@@ -0,0 +1,155 @@
+# Build stage 0 `builder`:
+# Extract Elasticsearch artifact
+################################################################################
+
+FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
+
+ARG PATCHFILE="https://raw.githubusercontent.com/ppc64le/build-scripts/master/e/elasticsearch/elasticsearch_8.1.1.patch"
+ARG ELASTIC_VERSION=${1:-v8.2.0}
+ARG MINIMUM_VERSION="v8.0.1"
+
+# `tini` is a tiny but valid init for containers. This is used to cleanly
+# control how ES and any child processes are shut down.
+#
+# The tini GitHub page gives instructions for verifying the binary using
+# gpg, but the keyservers are slow to return the key and this can fail the
+# build. Instead, we check the binary against the published checksum.
+RUN set -eux ; \
+    tini_bin="" ; \
+    case "$(arch)" in \
+        aarch64) tini_bin='tini-arm64' ;; \
+        x86_64)  tini_bin='tini-amd64' ;; \
+        ppc64le) tini_bin='tini-ppc64le' ;; \
+        *) echo >&2 ; echo >&2 "Unsupported architecture $(arch)" ; echo >&2 ; exit 1 ;; \
+    esac ; \
+    curl --retry 10 -S -L -O https://github.com/krallin/tini/releases/download/v0.19.0/${tini_bin} ; \
+    curl --retry 10 -S -L -O https://github.com/krallin/tini/releases/download/v0.19.0/${tini_bin}.sha256sum ; \
+    sha256sum -c ${tini_bin}.sha256sum ; \
+    rm ${tini_bin}.sha256sum ; \
+    mv ${tini_bin} /bin/tini ; \
+    chmod +x /bin/tini
+
+WORKDIR /opt
+
+RUN yum install -y wget git unzip \
+    && yum clean all
+
+RUN if [[ $ELASTIC_VERSION > $MINIMUM_VERSION ]]; then echo "$ELASTIC_VERSION>$MINIMUM_VERSION" && wget https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.2%2B8/OpenJDK17U-jdk_ppc64le_linux_hotspot_17.0.2_8.tar.gz && tar -C /usr/local -xzf OpenJDK17U-jdk_ppc64le_linux_hotspot_17.0.2_8.tar.gz && ln -sf /usr/local/jdk-17.0.2+8/bin/java /usr/bin/ && rm -f OpenJDK17U-jdk_ppc64le_linux_hotspot_17.0.2_8.tar.gz; else wget https://github.com/AdoptOpenJDK/openjdk15-binaries/releases/download/jdk15u-2021-01-22-02-31/OpenJDK15U-jdk_ppc64le_linux_hotspot_2021-01-22-02-31.tar.gz && tar -C /usr/local -xzf OpenJDK15U-jdk_ppc64le_linux_hotspot_2021-01-22-02-31.tar.gz && export JAVA_HOME=/usr/local/jdk-15.0.2+7/ && export JAVA15_HOME=/usr/local/jdk-15.0.2+7/ && export PATH=$PATH:/usr/local/jdk-15.0.2+7/bin && ln -sf /usr/local/jdk-15.0.2+7/bin/java /usr/bin/; fi
+
+RUN git clone --single-branch --branch $ELASTIC_VERSION https://github.com/elastic/elasticsearch.git && \
+    cd elasticsearch && git checkout $ELASTIC_VERSION && \
+    wget https://raw.githubusercontent.com/ppc64le/build-scripts/master/e/elasticsearch/elasticsearch_$PATCHFILE.patch && \
+    git apply elasticsearch_$PATCHFILE.patch && \
+    ./gradlew :distribution:archives:linux-ppc64le-tar:assemble --parallel && \
+    mv ./distribution/archives/linux-ppc64le-tar/build/distributions/elasticsearch-$PATCHFILE-SNAPSHOT-linux-ppc64le.tar.gz /opt
```

Comment:
Should it be :
`./distribution/archives/linux-ppc64le-tar/build/distributions/elasticsearch-*-SNAPSHOT--linux-ppc64le.tar.gz /opt`